jest-transformer.js

```diff
@@ -1,2 +1,11 @@
 const babelPreset = require(`babel-preset-gatsby-package`)()
-module.exports = require(`babel-jest`).createTransformer(babelPreset)
+module.exports = require(`babel-jest`).createTransformer({
```

Comment:
is there a reason to not use the `ts-jest` transformer directly? I can imagine a number of reasons why, just curious

Reply:
I can't really explain it. But ts-jest wasn't allowing us to mix commonjs and module import/exports. Builds would be fine and all, but tests would break and basically complain that things didn't exist because the require didn't resolve to the same as the export.

And honestly this makes more sense anyhow. We are using babel for js files, we are using babel for ts files, why not use it for tests too.

Reply:
nice, that makes sense. eventually using project references and the module name mapper, you could build all typescript projects at once using `tsc —build` which is the official typescript reccomendation for large, multi-module projects. ts-jest can honor this if you tweak moduleNameMapper right, which means you can execute jest from the root and resolve all local interdependencies without running `tsc` at all.

Reply:
Yes! That is potentially a  goal once we achieve 100% transition